cfe_internal/enterprise/federation/federation.cf

```diff
@@ -153,12 +153,20 @@ bundle agent transport_user
           "$(ssh_known_hosts)",
           "$(ssh_config)"
         };
+      "ssh_home_cil_file" string => "/tmp/cftransport_ssh_home_t.cil";
+
   classes:
     enabled::
-      "selinux_enabled"
-        expression => strcmp("Enforcing", execresult("getenforce", useshell));
-      "missing_ssh_context"
-        expression => not( or(
+      _stdlib_path_exists_getenforce::
+        "selinux_enabled"
+          expression => strcmp("Enforcing", execresult("$(paths.getenforce)", useshell));
```

Comment:
Would be better to use the `selinuxenabled` tool which just exits with 0 or 1 accordingly. But this is OK too.

Reply:
Ok. Good to know.